tutorials/packaging/create-your-first-snap/create-your-first-snap.md

```diff
@@ -50,11 +50,17 @@ Survey
 ## Getting started
 Duration: 1:00
 
-This tutorial is based on Ubuntu (16.04 LTS or greater) and the `snapcraft` APT package.
+### Setup a development environment
+This tutorial requires an Ubuntu 16.04 based system, either:
```

Comment:
We normally write the full name of the distro when we first use it, eg. `This tutorial requires an Ubuntu 16.04 LTS (Xenial) based system, such as:`